doc/reference/peripherals/kscan.rst

```diff
@@ -0,0 +1,32 @@
+.. _kscan:
+
+
+KSCAN
+#####
+
+Overview
+********
+The kscan driver (keyboard scan matrix) is used for detecting a key press in a
+laptop keyboard.
```

Comment:
is this exclusive to a laptop (builtin)  keyboard?  Or for a connected (USB or PS/2) keyboard too? I think the word "laptop" should be changed to "connected" or simply deleted.

Reply:
It can also be used to scan any keyboard like 4x4 matrixes in ATMs, alarm systems, joysticks.